sycl/test/check_device_code/matrix/matrix-nvptx-bf16-test.cpp

```diff
@@ -0,0 +1,279 @@
+// REQUIRES: gpu, cuda
+
+// RUN: %clangxx -fsycl-device-only -fsycl-targets=nvptx64-nvidia-cuda -Xsycl-target-backend --cuda-gpu-arch=sm_80 -DSYCL_EXT_ONEAPI_MATRIX=3 -S -Xclang -emit-llvm %s -o -| FileCheck %s
+
+#include <CL/sycl.hpp>
+
+using namespace sycl;
+using namespace sycl::ext::oneapi::experimental::matrix;
+
+constexpr int stride = 16;
+
+int main() {
+
+  buffer<uint16_t, 1> bufA(nullptr, range<1>(1));
+  buffer<uint16_t, 1> bufB(nullptr, range<1>(1));
+  buffer<float, 1> bufC(nullptr, range<1>(1));
+  buffer<float, 1> bufD(nullptr, range<1>(1));
+
+  queue q;
+
+  q.submit([&](handler &cgh) {
+    auto accC = bufC.get_access<access::mode::read_write>(cgh);
+    auto accA = bufA.get_access<access::mode::read_write>(cgh);
+    auto accB = bufB.get_access<access::mode::read_write>(cgh);
+    auto accD = bufD.get_access<access::mode::read_write>(cgh);
+
+    cgh.parallel_for<class row_row_m16n16k16>(
+        nd_range<2>({1, 32}, {1, 32}), [=
+    ](nd_item<2> item) [[sycl::reqd_work_group_size(1, 1, 32)]] {
+          sycl::sub_group sg = item.get_sub_group();
+
+          joint_matrix<float, matrix_use::accumulator, 16, 16,
+                       matrix_layout::row_major>
+              sub_c;
+
+          joint_matrix<uint16_t, matrix_use::a, 16, 16,
+                       matrix_layout::row_major>
+              sub_a;
+
+          joint_matrix<uint16_t, matrix_use::b, 16, 16,
+                       matrix_layout::row_major>
+              sub_b;
+
+          // CHECK: tail call { float, float, float, float, float, float, float,
+          // float } @llvm.nvvm.wmma.m16n16k16.load.c.row.stride.f32.p1f32(float
+          // addrspace(1)* %add.ptr.i, i32 16) #{{.*}}
```

Comment:
I may be wrong, but I do not think this is how to do same-line check on multiple lines. I believe it's supposed to be either

```suggestion
          // CHECK: tail call { float, float, float, float, float, float, float,\
          // CHECK: float } @llvm.nvvm.wmma.m16n16k16.load.c.row.stride.f32.p1f32(float\
          // CHECK: addrspace(1)* %add.ptr.i, i32 16) #{{.*}}
```
or
```suggestion
          // CHECK: tail call { float, float, float, float, float, float, float,
          // CHECK-SAME: float } @llvm.nvvm.wmma.m16n16k16.load.c.row.stride.f32.p1f32(float
          // CHECK-SAME: addrspace(1)* %add.ptr.i, i32 16) #{{.*}}
```

Reply:
Yes you're right, it previously wasn't working correctly.  Thanks.

The second suggested change works (first one doesn't seem to), but in the end I decided to switch back to a single line CHECK.  I think it leaves less room for errors and I don't see any real advantage of splitting it over several lines.